pkg/ruptela/conver_signal_funcs_gen.go

```diff
@@ -14,28 +15,56 @@ import (
 // Vehicle.Chassis.Axle.Row1.Wheel.Left.Tire.Pressure: Tire pressure in kilo-Pascal.
 // Unit: 'kPa'
 func ToChassisAxleRow1WheelLeftTirePressure0(originalDoc []byte, val string) (float64, error) {
-	return Convert960(val)
+	psi, err := Convert960(val)
+	if err != nil {
+		return 0, err
+	}
+
+	kPa := ConvertPSIToKPa(psi)
+	kPa, _ = strconv.ParseFloat(fmt.Sprintf("%.2f", kPa), 64)
```

Comment:
What is the point of converting this value to a string and then back to a float? This is vert inefficient.

Reply:
1. I don't think we gain anything from making our data less precise.
2. For the future things math operations are always significantly cheaper then using strings.

Reply:
Ok, removing rounding